cfme/tests/containers/test_pods_prop_data_integrity.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+import pytest
+from cfme.fixtures import pytest_selenium as sel
+from cfme.containers.pod import Pod, list_tbl
+from utils import testgen
+from utils.version import current_version
+
+
+pytestmark = [
+    pytest.mark.uncollectif(
+        lambda: current_version() < "5.6"),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1)]
+pytest_generate_tests = testgen.generate(
+    testgen.container_providers, scope="function")
+
+# CMP-9934
+
+
+def test_summary_properties_validation(provider):
+    """
+          This test verifies that the number of running containers in the status summary table
+          is the same number that appears in the Relationships table
```

Comment:
Yeah. Maybe you should sum together the `running`, `terminated` and `waiting` to get the real total -- `obj.summary.relationships.containers.value`